packages/dds/tree/src/core/forest/forest.ts

```diff
@@ -5,7 +5,7 @@
 
 import { assert } from "@fluidframework/core-utils/internal";
 
-import type { Listenable } from "../../events/index.js";
+import type { Listenable } from "@fluidframework/core-utils";
```

Comment:
Probably merge with the import of `assert` above, same in other places where both are imported.